### ISSUE TITLE
Regression in HEAD with acli refresh for Site Factory applications

### ISSUE THREAD

**Issue by anavarre** [acquia/cli#268]
This looks to be a regression in HEAD. This is happening with Site Factory applications, upon selecting a DB to refresh.

```
$ acli refresh
Could not find a local Drupal project. Looked for docroot/index.php in current and parent directories
Could not find a git repository in the current directory
Would you like to clone a project into the current directory? yes
Using Cloud Application MY SITE FACTORY APP 
Choose a Cloud Platform environment to copy from:
  [0] 01dev (vcs: develop)
  [1] 01_test (vcs: composer-refactor-070920)
 > 01dev (vcs: develop)
    ✔ Cloning git repository from the Cloud Platform
Choose a database to copy:
  [0 ] jxr136
  [1 ] jxr141
  [2 ] jxr276
  [3 ] jxr281
  [4 ] jxr286
  [5 ] jxr291
 > 3
    ⌛ Importing Drupal database copy from the Cloud Platform...
PHP Fatal error:  Uncaught TypeError: Argument 3 passed to Acquia\Cli\Command\RefreshCommand::createAndImportRemoteDatabaseDump() must be of the type string, null given, called in /home/ide/cli/src/Command/RefreshCommand.php on line 450 and defined in /home/ide/cli/src/Command/RefreshCommand.php:193
Stack trace:
#0 /home/ide/cli/src/Command/RefreshCommand.php(450): Acquia\Cli\Command\RefreshCommand->createAndImportRemoteDatabaseDump(Object(AcquiaCloudApi\Response\EnvironmentResponse), Object(stdClass), NULL, 'sitefactorydb5557...', Object(Closure))
#1 /home/ide/cli/src/Command/RefreshCommand.php(96): Acquia\Cli\Command\RefreshCommand->importRemoteDatabase(Object(AcquiaCloudApi\Response\EnvironmentResponse), Object(stdClass), Object(Closure))
#2 /home/ide/cli/vendor/symfony/console/Command/Command.php(258): Acquia\Cli\Command\RefreshCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /home/ide/cli/src/Command/CommandBase.php(250): Symfony\Component\Console in /home/ide/cli/src/Command/RefreshCommand.php on line 193
```

Note: I did not observe this issue with ACE with multiple databases.

**Comment by grasmash** [comment]
Yeah I think we need to revert https://github.com/acquia/cli/pull/261. I removed a workaround for an upstream issue that was marked as "Resolved," but likely it's not actually resolved for ACSF.

**Comment by grasmash** [comment]
https://github.com/acquia/cli/pull/269

**Comment by grasmash** [comment]
@anavarre Please test to see if this has resolved the issue. I've re-opened https://github.com/acquia/cli/issues/246

**Comment by grasmash** [comment]
K this has been resolved and I also found the correct fix for #246.

**Comment by anavarre** [comment]
It does seem to have been resolved but #246 is still very much happening in my testing.
